examples/example_step.py

```diff
@@ -11,7 +11,7 @@
 obs = pd.read_csv("data/head_nb1.csv", index_col=0, parse_dates=True,
                   squeeze=True)
 # add 10 cm to the series from 2007
```

Comment:
nitpicking here, but should probably change the comment too.

Reply:
#180